src/feature/relay/dns.c

```diff
@@ -127,6 +129,8 @@ static uint64_t n_ipv6_requests_made = 0;
 static uint64_t n_ipv6_timeouts = 0;
 /** Global: Do we think that IPv6 DNS is broken? */
 static int dns_is_broken_for_ipv6 = 0;
+/** Has IPv6 DNS failed ever? */
+static int dns_was_broken_for_ipv6 = 0;
```

Comment:
Are this comment and variable name accurate?

The code resets IPv6 DNS after 24 hours, so it would be helpful to say "recent".